src/Link.php

```diff
@@ -110,7 +110,7 @@ public function withHref(string|Stringable $href): self
      */
     public function withRel(string $rel): self
     {
-        if (! is_string($rel) || empty($rel)) {
+        if (! is_string($rel) || ($rel === '' || $rel === '0')) {
```

Comment:
just quick look, on `||`, parentheses on the right seems not needed

```suggestion
        if (! is_string($rel) || $rel === '' || $rel === '0') {
```

Reply:
@samsonasik This is going into 3.0 though, so it doesn't matter.

Reply:
new test seems needed then